<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="setup_dockwidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>260</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>260</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Setup</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="sensors_label">
         <property name="text">
          <string>Sensor Combination:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="sensors_cbox">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D LiDAR - 3D LiDAR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D LiDAR - RGB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3D LiDAR - RGB-D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RGB-D - RGB-D</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RGB-D - RGB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RGB - RGB</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="algo_label">
         <property name="text">
          <string>Calibration Algorithm:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="algo_cbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Plane matching</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Line matching</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Trajectory matching</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="irx_label">
         <property name="text">
          <string>iRx:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="irx_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="iry_label">
         <property name="text">
          <string>iRy:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="iry_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="irz_label">
         <property name="text">
          <string>iRz:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QDoubleSpinBox" name="irz_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="itx_label">
         <property name="text">
          <string>iTx:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="itx_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="ity_label">
         <property name="text">
          <string>iTy:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="ity_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="itz_label">
         <property name="text">
          <string>iTz:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QDoubleSpinBox" name="itz_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-359.990000000000009</double>
         </property>
         <property name="maximum">
          <double>359.990000000000009</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="med_label">
         <property name="text">
          <string>MED:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="med_sbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>-10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="load_rlog_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Load Rawlog</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start_calib_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start Calibration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="continue_calib_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Continue</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="save_calib_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Calibration</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="obs_dockwidget">
   <property name="minimumSize">
    <size>
     <width>88</width>
     <height>107</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Observations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
